nests.json

```diff
@@ -346,6 +346,17 @@
       "status": "stable",
       "category": "scientific",
       "readme": "https://raw.githubusercontent.com/Libraries4U/Box2D/main/README.md"
-    }
+    },
+    {
```

Comment:
Could you put it after Skylar package? I think it would be easier to find Skylar related packages.

Reply:
Done

Reply:
@klugier 

I think we can do that after merging the changes. The real question to me is the directory structure. Since this is a demo app, it should be placed in `examples` directory. Do we agree on this or do you beg to differ? (This part isn't clear in the instructions, so I am asking again)

Reply:
I agree with you Ismail, I'm working on moving all source code to a src dir and moving all the app in examples dir. 
I will also add a license. Will notice you when it's rdy to merge

Reply:
OK, thanks. I'll be waiting for your notification.

Reply:
Done, however the new files hierachy lead to this : https://i.imgur.com/6WgNNy9.png

Reply:
@Xemuth 
Hmm. since this is not a module/ctrl, and we don't have clear info on this specific case, maybe we should stick to the app structure. (We'll fix it if it needs to be fixed later.)
 
 Could you just move the dirs in the `examples/SkylarkUploaders` up to the root, and test it on your local machine?
(And, sorry for the inconvenience.)

E.g.:

```
 `SyklarkUploaders/src`
 `SkylarkUploaders/views`
 `SkylarkUploaders/resources`
 `SkylarkUploaders/SkylarkUploaders.upp`
 `SkylarkUploaders/README.md`
 `SkylarkUploaders/LICENSE`
```

Reply:
should be good now

Reply:
OK, thanks.